tests/cache/models/samples/get.js

```diff
@@ -582,3 +511,123 @@ describe('tests/cache/models/samples/get.js, ' +
     .end(done);
   });
 });
+
+describe('tests/cache/models/samples/get.js, ' +
+ 'Basic Get with cacheGetSamplesWildcard flag on', () => {
+  let token;
+  const s1s2a1 = '___Subject1.___Subject2|___Aspect1';
+  const s1s2a2 = '___Subject1.___Subject2|___Aspect2';
+  const s1s3a1 = '___Subject1.___Subject3|___Aspect1';
+
+  before((done) => {
+    tu.toggleOverride('cacheGetSamplesByNameWildcard', true);
+    tu.toggleOverride('enableRedisSampleStore', true);
+    tu.createToken()
+    .then((returnedToken) => {
+      token = returnedToken;
+      done();
+    })
+    .catch(done);
+  });
+
+  before(rtu.populateRedis);
+  after(rtu.forceDelete);
+
+  after(() => {
+    tu.toggleOverride('cacheGetSamplesByNameWildcard',
+      cacheGetSamplesByNameWildcard);
+    tu.toggleOverride('enableRedisSampleStore',
+      enableRedisSampleStore);
+  });
+
+  it('get with wildcard should cache response', (done) => {
+    api.get(`${path}?name=___Subj*`)
+    .set('Authorization', token)
+    .expect(constants.httpStatus.OK)
+    .end((err, res) => {
+      if (err) {
+        return done(err);
+      }
+
+      redisCache.get('___Subj*', (cacheErr, reply) => {
+        if (cacheErr || !reply) {
+          return done(cacheErr);
```

Comment:
If cacheErr is false-y here, it means we have `!reply`, but we're returning `done(cacheErr)` (with an empty `cacheErr`, and calling done with nothing is treated as test success, not test failure.